### PR TITLE
MCKIN-19840 EOC (End of course Journal) - None of the strings are translated in any language

### PR DESCRIPTION
**Pull request by wasifarbisoft**
https://edx-wiki.atlassian.net/browse/MCKIN-19840

### CODE REVIEW

**Comment by musmanmalik** [comment]
@xitij2000 Please review this PR.

File: eoc_journal/translations/config.yaml [hunk 1]
```diff
@@ -12,9 +12,6 @@ dummy_locales:
 # Directories we don't search for strings.
 ignore_dirs:
     - tests
-    - eoc_journal/conf
-    - eoc_journal/translations
-    - eoc_journal/public/js/translations
```
Comment (xitij2000) [inline eoc_journal/translations/config.yaml:17]: Why are these being removed?

File: eoc_journal/eoc_journal.py [hunk 2]
```diff
@@ -218,17 +220,27 @@ def student_view(self, context=None):
 
         fragment = Fragment()
         fragment.add_content(
-            loader.render_template("templates/eoc_journal.html", context)
+            loader.render_django_template('templates/eoc_journal.html', context,
+                                                           i18n_service=self.i18n_service)
```
Comment (xitij2000) [inline eoc_journal/eoc_journal.py:224]: This formatting doesn't look right.

**Comment by xitij2000** [comment]
@wasifarbisoft Please look at the issues raised by the CI and fix them.

File: eoc_journal/translations/config.yaml (hunk 1 above)
Comment (wasifarbisoft) [reply eoc_journal/translations/config.yaml]: reverted this change, perhaps not needed at this time.

File: eoc_journal/eoc_journal.py (hunk 2 above)
Comment (wasifarbisoft) [reply eoc_journal/eoc_journal.py]: Fixed!

File: eoc_journal/templates/eoc_journal.html [hunk 3]
```diff
@@ -145,7 +145,7 @@ <h4>{{ pdf_report_link_heading }}</h4>
   {% if display_key_takeaways %}
   <div class="eoc-key-takeaways">
     <div class="title">
-      <h4>Key Takeaways</h4>
+      <h4>{% trans "Key Takeaways" %}</h4>
```
[the remark this answers is not in the document]
Comment (wasifarbisoft) [reply eoc_journal/templates/eoc_journal.html]: Added the force_escape filter. 
Thanks for the tip.

**Comment by wasifarbisoft** [comment]
@xitij2000 , @taniwha ,
I have made the changes and updated the pull request accordingly,
CI Issues are now resolved.

File: eoc_journal/eoc_journal.py [hunk 4]
```diff
@@ -30,17 +31,14 @@
...
 
 try:
     from django.contrib.auth.models import User
 except ImportError:
     User = None  # pylint: disable=C0103
 
 
-loader = ResourceLoader(__name__)
-
-
```
Comment (xitij2000) [inline eoc_journal/eoc_journal.py:43]: What is the reason for this change? Please undo it since it creates an unnecessary diff.

**Comment by xitij2000** [comment]
@wasifarbisoft Please also fix the PR title and description..

**Comment by wasifarbisoft** [comment]
@xitij2000 I have made the changes you suggested!

File: requirements.txt [hunk 5]
```diff
@@ -1,4 +1,4 @@
-git+https://github.com/edx/xblock-utils.git@v1.0.2#egg=xblock-utils==1.0.2
+git+https://github.com/edx/xblock-utils.git@v1.1.1#egg=xblock-utils==1.1.1
```
Comment (xitij2000) [inline requirements.txt:1]: We can just switch to the pypi version now. 
```suggestion
xblock-utils==1.2.2
```

File: requirements.txt [hunk 6]
```diff
@@ -1,4 +1,4 @@
-git+https://github.com/edx/xblock-utils.git@v1.0.2#egg=xblock-utils==1.0.2
+git+https://github.com/edx/xblock-utils.git@v1.2.2#egg=xblock-utils==1.2.2
```
Comment (xitij2000) [inline requirements.txt:1]: I recommended this since this block is available on PyPI now, and that makes for a much quicker install. 
```suggestion
xblock-utils==1.2.2
```
